Pages/product.css

```diff
@@ -0,0 +1,183 @@
+body {
+    min-height: auto;
+    overflow-x: hidden;
+}
+
+main {
+    display: flex;
+}
+
+.main-section {
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+hr {
+    width: 50%;
+}
+
+.section-filter {
+    max-width: 25vw;
+    border-right: 0.2rem solid var(--pink-nav);
+    position: sticky;
+    top: 6.5vh;
+    left: 0;
+    overflow: auto;
+    height: 100%;
+    padding: 1rem;
+    overflow: hidden;
+}
+
+.section-filter div {
+    display: flex;
+    align-items: center;
+    width: 25vw;
+}
+
+.header-filter {
+    color: var(--gray-mild);
+}
+
+.sub-header-filter {
+    color: var(--gray-mild);
+    font-weight: 800;
+}
+
+.filters-applied {
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+    gap: 1rem;
+}
+
+.switch {
+    position: relative;
+    display: inline-block;
+    width: 3rem;
+    height: 1.5rem;
+}
+
+.slider {
+    background-color: var(--gray-mild);
+    -webkit-transition: 0.4s;
+    transition: 0.4s;
+}
+
+.slider:before {
+    height: 1.1rem;
+    width: 1.1rem;
+    left: 0.3rem;
+    bottom: 0.2rem;
+    background-color: var(--white);
+    transition: 0.4s cubic-bezier(0.5, 0.5, 0.5, 1);
+}
+
+input:checked+.slider {
+    background-color: var(--pink-nav);
+}
+
+input:focus+.slider {
+    box-shadow: 0 0 0.2rem var(--gray-mild);
+}
+
+input:checked+.slider:before {
+    -webkit-transform: translateX(1rem);
+    -ms-transform: translateX(1.3rem);
+    transform: translateX(1.3rem);
+}
+
+.styled-slider {
+    -webkit-appearance: none;
+    width: 100%;
+    cursor: pointer;
+}
+
+.styled-slider:focus {
+    outline: none;
+}
+
+.styled-slider::-webkit-slider-runnable-track {
+    background: var(--pink-nav);
+    height: 0.2rem;
+}
+
+.price-range {
+    display: flex;
+    justify-content: space-between;
+    color: var(--gray-mild);
+}
+
+.checkbox {
+    color: var(--gray-dominant);
+    padding: 0.2rem;
+}
+
+.radio {
+    color: var(--gray-dominant);
+    padding: 0.2rem;
+}
+
+.input-desc {
+    padding: 0 1rem;
+}
+
+.styled-slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    background: var(--white);
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 50%;
+    border: 0.2rem solid var(--pink-nav);
+    margin-top: -0.70rem;
+}
+
+.section-card {
+    margin-top: 3.5rem;
+    padding: 2rem;
+}
+
+.path {
+    color: var(--gray-light);
+}
+
+.container-card {
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    justify-content: space-evenly;
+}
+
+.card {
+    margin: 3rem 1rem;
+    border-radius: 0.25rem;
+    box-shadow: 0.1rem 0.1rem 0.5rem var(--gray-mild)
```

Comment:
use px for units less than 1rem

Reply:
Is it mandatory??
and why should we use px??

Reply:
Not mandatory but when your viewport changes rem value in px changed then you don't want these small size value to change. It's ok if you want them to change on changing viewport.